src/app/garfile/home/post.controller.js

```diff
@@ -28,10 +28,13 @@ module.exports = (req, res) => {
           const parsedResponse = JSON.parse(apiResponse);
           if (parsedResponse.hasOwnProperty('message')) {
             // API returned error
-            res.render('app/garfile/manifest/index', { errors: [parsedResponse], cookie });
+            res.render('app/garfile/manifest/index', {
+              errors: [parsedResponse],
+              cookie,
+            });
           } else {
-          cookie.setGarId(parsedResponse.garId);
-          cookie.setGarStatus(garStatus);
+            cookie.setGarId(parsedResponse.garId);
+            cookie.setGarStatus(garStatus);
           }
           if (req.body.garoption === 'Create a GAR online') {
```

Comment:
Nigel, When I look at https://jhvxrb3.egar-staging.homeoffice.gov.uk/version3/create-gar-landing, the recommended text for radio button is "Upload a file" and "Online". But the text we have here is "Create a GAR from file" and "Create a GAR from file". Is there a reason the text is not different from prototype?